rpcs3/rpcs3qt/msg_dialog_frame.cpp

```diff
@@ -110,10 +117,82 @@ void msg_dialog_frame::Create(const std::string& msg)
 	m_dialog->setLayout(layout);
 
 	connect(m_dialog, &QDialog::rejected, [=] {if (!type.disable_cancel) { on_close(CELL_MSGDIALOG_BUTTON_ESCAPE); }});
+	connect(m_dialog, &QDialog::destroyed, [=] {if (--dialog_count <= 0) dialog_nr = 0;});
 
 	//Fix size
 	m_dialog->setFixedSize(m_dialog->sizeHint());
+
+	// order compile-dialogs on screen
+	if (qstr(msg).contains("Compiling PPU module"))
```

Comment:
@Megamouse Why did you do this? `msg` is `std::string`, you don't need to construct a `QString` from it just to search a string. use `std::find`.